pandas/util/windows.py

```diff
@@ -0,0 +1,151 @@
+"""
+This module implements clipboard handling on Windows using ctypes.
+"""
+import time
+import contextlib
```

Comment:
i have moved all files related to pyperclip to pandas/util/clipboard. Also i have deleted the clipboard file and retained the __init__.py from the original distribution. This negates the need to replace references as the module is still being access by the address pandas.util.clipboard